tests/HUBComponentModelBuilderTests.m

```diff
@@ -592,4 +592,88 @@ - (void)testCopying
     XCTAssertEqualObjects(copiedCustomImageDataBuilder.placeholderIconIdentifier, @"customPlaceholder");
 }
 
+- (void)testBuildersForChildrenInGroupWhenAddingChildBuilder
+{
+    NSString * const firstChildModelIdentifier = @"firstIdentifier";
+    NSString * const secondChildModelIdentifier = @"secondIdentifier";
+    NSString * const firstChildGroupIdentifier = @"firstGroup";
+    NSString * const secondChildGroupIdentifier = @"secondGroup";
+
+    id<HUBComponentModelBuilder> const childBuilder = [self.builder builderForChildWithIdentifier:firstChildModelIdentifier];
+    childBuilder.groupIdentifier = firstChildGroupIdentifier;
+    id<HUBComponentModelBuilder> const childBuilder2 = [self.builder builderForChildWithIdentifier:secondChildModelIdentifier];
+    childBuilder2.groupIdentifier = secondChildGroupIdentifier;
+
+    NSArray *buildersForChildrenInGroup = [self.builder buildersForChildrenInGroupWithIdentifier:firstChildGroupIdentifier];
```

Comment:
You're only verifying `firstChildGroupIdentifier`. How about verifying the second one too?